wherehows-web/tests/test-helper.js

```diff
@@ -2,6 +2,15 @@ import Application from '../app';
 import config from '../config/environment';
 import { setApplication } from '@ember/test-helpers';
 import { start } from 'ember-qunit';
+import { registerDeprecationHandler } from '@ember/debug';
+
+// Suggestion to keep this here until we are ready to handle the deprecation to use definePRoperty for
+// computed properties. Otherwise, tests and build logs becoming overwhelming to look at
+registerDeprecationHandler(function(message, { id }, next) {
+  if (!message.includes('defineProperty')) {
```

Comment:
I would say this should be shown at least once, totally hiding and I can almost guarantee it will never get dealt with because it will be forgotten :(. The logs are meant to nag into action. Also create an issue to track this